### PR TITLE
Sdc expansion

### PR DESCRIPTION
**Pull request by Krunal-Thakkar**
# Description
Added Functions for the Gateway Installer Operations

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/814 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
image

### CODE REVIEW

File: types/v1/types.go [hunk 1]
```diff
@@ -1304,3 +1305,83 @@ type NFSExportModify struct {
...
+	StatusCode int    `json:"httpStatusCode,omitempty"`
+	ErrorCode  int    `json:"errorCode,omitempty"`
+}
+
+// MDMTopologyParam defines struct for Validate MDM Topology
+type MDMTopologyParam struct {
+	MdmIps                []string `json:"mdmIps"`
+	MdmUser               string   `json:"mdmUser"`
+	MdmPassword           string   `json:"mdmPassword"`
+	SecurityConfiguration struct {
```
Comment (VamsiSiddu-7) [inline types/v1/types.go:1340]: Instead of  creating the struct inline can you create a  new separate struct and place the name of the struct here.

Comment (Krunal-Thakkar) [reply types/v1/types.go:1340]: @VamsiSiddu-7 I created inline because no use separate use of that inline struct.

Comment (VamsiSiddu-7) [reply types/v1/types.go:1340]: But i am thinking it is good for readability if we have a separate struct.

**Review by khareRajshree — APPROVED** [review]

**Review by VamsiSiddu-7 — APPROVED** [review]

**Review by rounak-adhikary — APPROVED** [review]
lgtm

**Pull request merged**